src/broker/runat.c

```
@@ -433,6 +433,7 @@ static int runat_push (struct runat *r,
         if (!(entry = runat_entry_create (name)))
             return -1;
         if (zhashx_insert (r->entries, name, entry) < 0) {
+            errno = EEXIST;
```

same here, just ignore the error